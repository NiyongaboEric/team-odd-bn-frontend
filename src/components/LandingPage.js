import React from 'react';
import { Link } from 'react-router-dom';
import '../assets/css/style.css';
import logo from '../assets/images/logo.png';

const landingPage = () => (
  <div className="row">
    <div className="column-left">
      <div className="main-logo">
        <div className="logo">
          <img src={logo} alt="logo" />
        </div>
        <div className="logo-name">
          <a href="#home">Barefoot</a>
        </div>

      </div>
      <div className="main-title">
        <h4>
Making company travel and
          <br />
accomodation easy and convenient
        </h4>
        <Link to="signin">
          <button className="button">Login to Get Started</button>

        </Link>
      </div>
    </div>
    <div className="column-right" />
  </div>
);
export default landingPage;
