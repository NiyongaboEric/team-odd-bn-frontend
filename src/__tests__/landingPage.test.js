import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../components/LandingPage';

describe('Test Signout page ', () => {
  it('should render', () => {
    const wrapper = shallow(<LandingPage />);
    expect(shallow(<LandingPage />)).toMatchSnapshot();
  });
});
