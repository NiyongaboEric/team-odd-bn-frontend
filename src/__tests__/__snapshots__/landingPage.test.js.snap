// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Signout page  should render 1`] = `ShallowWrapper {}`;
